features/shipping/applying_shipping_method_rules/viewing_available_shipping_methods_based_on_channel_as_an_logged_in_customer.feature

```diff
@@ -0,0 +1,34 @@
+@applying_shipping_method_rules
+Feature: Viewing available shipping methods based on channel
```

Comment:
same

Reply:
can we merge these two features? both are related with available shipping methods